java-extensions/iceberg-metadata-reader/src/main/java/com/starrocks/connector/iceberg/TableFileIOCache.java

```diff
@@ -18,10 +18,13 @@
 import com.google.common.cache.CacheBuilder;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.io.FileIO;
+import org.slf4j.LoggerFactory;
 
 import java.util.concurrent.TimeUnit;
 
 public class TableFileIOCache {
+    public static final org.slf4j.Logger LOG = LoggerFactory.getLogger(AbstractIcebergMetadataScanner.class);
```

Comment:
why using slf4j

Reply:
I don't know the difference. But I've tested that it works well with log4j2.properties.